src/Kernel/Magic/EstimateMagic.cs

```diff
@@ -28,9 +28,40 @@ public class EstimateMagic : AbstractMagic
         /// </summary>
         public EstimateMagic(ISymbolResolver resolver) : base(
             "estimate",
-            new Documentation
-            {
-                Summary = "Runs a given function or operation on the ResourcesEstimator target machine."
+            new Documentation {
```

Comment:
Exceedingly minor, but elsewhere you've had the braces on the next line.
```suggestion
            new Documentation
            {
```